### ISSUE TITLE
Rename Licence in the Create and View pages

### ISSUE THREAD

**Issue by annikaLiving** [bcgov/ckan-ui#304]
License needs to be spelt "Licence" from the view and create pages.

**Comment by SteveChapmanBCDX** [comment]
This issue keeps cropping up.  Perhaps we should do an audit of the code base to standardize this term.

**Comment by TerryLanktree** [comment]
I would not have thought this would require so much time.  I looked up what the difference is, and when to use each.  Noun vs. verb, but it appears to me to be used as an adjective in the catalogue classic (unless it's a passive verb, I guess I should have paid more attention in grade 11 English).  Bottom line, it's ok to use Licence interchangeably with nouns and verbs in British English.  So, I would recommend we do a find/replace for every instance.  Change "license" to "licence".  I noticed this does not align with Cat Classic, so open for discussion.
